lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html

```diff
@@ -81,6 +83,7 @@ <h3>{{'PERMISSION_MANAGER.LABELS.DIRECT-PERMISSIONS' | translate }}</h3>
                 <button
                     mat-button
                     [matTooltip]="'PERMISSION_MANAGER.ACTION.DELETE' | translate"
+                    title="{{'PERMISSION_MANAGER.ACTION.DELETE' | translate}}"
```

Comment:
conflicts with the matTooltip, one of those should be removed

Reply:
noted and resolved